blueflood-core/src/main/java/com/rackspacecloud/blueflood/service/BluefloodServiceStarter.java

```diff
@@ -295,8 +303,10 @@ public void run() {
                     TimeUnit.MINUTES.toMillis(savePeriodMins));
         }
 
-        // has the side-effect of causing static initialization of Metrics, starting instrumentation reporting.
-        new RestartGauge(Metrics.getRegistry(), RollupService.class);
+        if (shouldInstantiateRestartGauge) {
```

Comment:
@izrik I had created a PR against your branch. https://github.com/rackerlabs/blueflood/pull/611

Pls try this and let me know if it looks okay. I had to settle for a lower version of powermock which is compatible with the version of mockito we have. May be I should have upgraded both. I didnt try that.

While I was looking into it, I felt like there is no need for all the methods in BluefloodServiceStarter to be static. Its a good idea to actually make all methods non-static, instantiate the class from main method and call the methods using that. May be for a different PR.